src/Utils/Hooks/useDebounce.ts

```diff
@@ -22,6 +25,9 @@ interface UseDebouncedValue<T> {
   settings?: any
 }
 
+/**
+ * @deprecated Use `import { useDebounce } from "use-debounce"` instead
+ */
```

Comment:
None of these debounces worked correctly; they would just delay the fetch and stack them. I think filters has a debounce internally that works that prevents this from being a problem.